ambari/src/test/resources/ambari-alert-notifications.md

```diff
@@ -0,0 +1,32 @@
+#Ambari Alert Notifications
```

Comment:
Why is this file in test/resources? Doesn't feel like the best place - probably need to think about how to expand our docs.